static/js/rendered_markdown.js

```diff
@@ -159,7 +159,8 @@ exports.update_elements = (content) => {
         if (isValid(timestamp)) {
             const text = $(this).text();
             const rendered_time = timerender.render_markdown_timestamp(timestamp, text);
-            $(this).text(rendered_time.text);
+            $(this).empty();
+            $(this).append(`<i class="fa fa-clock-o"></i>`, rendered_time.text);
```

Comment:
`append` still interprets the string `rendered_time.text` as HTML.

```suggestion
            $(this).text(rendered_time.text)
            $(this).prepend($("<i>", {class: "fa fa-clock-o"}));
```